server/tests/test_processors_pipeline.py

```diff
@@ -1,61 +0,0 @@
-import pytest
```

Comment:
the test didn't do nothing

a new test required mocks up to the level of websocker+redis - seemingly a sign of needed refactoring, we're not doing it in this PR